basement/Basement/From.hs

```diff
@@ -191,7 +192,9 @@ instance From (Maybe a) (Either () a) where
 instance From (CountOf ty) Int where
     from (CountOf n) = n
 instance From (CountOf ty) Word where
-    from (CountOf n) = from n
+    -- here it is ok to cast the underlying `Int` held by `CountOf` to a `Word`
```

Comment:
documentation talk about cast, probably 'convert' would be less confusing considering the Cast class